design_document.md

```diff
@@ -6,10 +6,7 @@ This document provides the common design features to be followed while developin
 
 Icons are available in 4 sizes : 16, 24, 32, 48. It is possibility that we might not have all sizes for all icons. Edit the number as per size chose, everything else will remain same.
```

Comment:
im curious as to how this file was changed in your branch. please revert this back

Reply:
The Markdown table was malformed, and would not render properly here on GitHub. I will still revert the change if you'd like.

Reply:
don't worry about it. i didn't realize there was a problem. good catch!